src/uCodes/ZSortBOSS.cpp

```diff
@@ -519,9 +519,9 @@ void ZSortBOSS_Lighting( u32 _w0, u32 _w1 )
 	for(u32 i = 0; i < num; i++) {
 		SPVertex & vtx = pVtx[i];
 
-		vtx.nx = ((s8*)DMEM)[(nsrs++)^3] * 0.00390625f;
```

Comment:
@Gillou68310 I just noticed that. Normal is signed char. Should not it be divided by 127 instead of 256?